<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonPropertiesWidget</class>
 <widget class="QWidget" name="CommonPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="ql_pH">
       <property name="text">
        <string>pH</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="FloatingValueLineEdit" name="qle_pH">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ql_ionicStrength">
       <property name="text">
        <string>Ionic strength (mM)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="FloatingValueLineEdit" name="qle_ionicStrength">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ql_conductivity">
       <property name="text">
        <string>Conductivity (S/m)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="FloatingValueLineEdit" name="qle_conductivity">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ql_bufferCapacity">
       <property name="text">
        <string>Buffer capacity (mM)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="FloatingValueLineEdit" name="qle_bufferCapacity">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="ql_systemEigenmobilities">
     <property name="text">
      <string>System eigenmobilities</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="qtbw_systemEigenzones"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatingValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/ui/elementaries//floatingvaluelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
